content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -3,17 +3,30 @@ Colombia
 ========
 
 .. |DIAN| replace:: :abbr:`DIAN (Dirección de Impuestos y Aduanas Nacionales)`
+.. |NIT| replace:: :abbr:`NIT (El Número de Identificación Tributaria)`
+.. _dian-certification: https://micrositios.dian.gov.co/sistema-de-facturacion-electronica/proceso-de-registro-y-habilitacion-como-facturador-electronico/
 
 Odoo's Colombian localization package provides accounting, fiscal, and legal features for databases
-in Colombia – such as chart of accounts, taxes, and electronic invoicing.
-
-In addition, a series of videos on the subject is also available. These videos cover how to start
-from scratch, set up configurations, complete common workflows, and provide in-depth looks at some
-specific use cases as well.
+in Colombia - such as chart of accounts, taxes, and electronic invoicing. The localization has the
+following `prerequisites
+<https://micrositios.dian.gov.co/sistema-de-facturacion-electronica/que-requieres-para-factura-electronicamente/>`_
+when using the `DIAN Own Software
+<https://micrositios.dian.gov.co/sistema-de-facturacion-electronica/como-puedes-facturar-electronicamente/>`_
+solution with Odoo:
+
+- Be registered in the `RUT
+  <https://www.dian.gov.co/tramitesservicios/tramites-y-servicios/tributarios/Paginas/RUT.aspx>`_
+  (Registro Único Tributario) with a valid |NIT|.
+- Have a valid digital signature certificate `approved by the ONAC
+  <https://onac.org.co/directorio-de-acreditados/>`_.
+- `Register and get enabled <dian-certification>`_ by completing the certification process required
```

Comment:
```suggestion
- `Register and get enabled <https://micrositios.dian.gov.co/sistema-de-facturacion-electronica/proceso-de-registro-y-habilitacion-como-facturador-electronico/>`_ by completing the certification process required
```

Reply:
@samueljlieber added this link so it can work and take you to the defined website